pkg/operator/k8sutil/prometheus_test.go

```diff
@@ -0,0 +1,21 @@
+package k8sutil
+
+import (
+	"testing"
+)
+
+func TestGetPrometheusRule(t *testing.T) {
+	var filePath = "../../../cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml"
```

Comment:
>
> Is this v14 version file in the branch?
>
Not yet. Wanted to know your thoughts before adding that change (maybe as a separate PR).
I was thinking of a config file to map these, rather than duplicating files.
>
> I'm not sure if the unit tests will work reliably with the relative path since the current directory might not always be the same.
>
How do we take care of this? Maybe absolute path (with GOPATH)?

Agree that Integration Tests should be a separate issue.

Reply:
> Not yet. Wanted to know your thoughts before adding that change (maybe as a separate PR).
I was thinking of a config file to map these, rather than duplicating files?

Yes, this seems like the best way to support ceph-mixins for different major versions of ceph. If multiple versions are in the container, the operator can install the appropriate version for the cluster being installed.